### DISCUSSION TITLE
[Unstable] Crash on starting campaign

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#15407] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

[crashreport (60).log](https://github.com/user-attachments/files/18204403/crashreport.60.log)

### Reproduction steps

start a campaign

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version
Unstable V1.8.0.0

**Comment by Regalis11** [comment]
> - [ ] My issue happened while using mods.
> Selected content packages: Vanilla (553389D), A Testbox (B0E35CC), DynamicEuropa (92C1558)

:baromodder:

Addressed in https://github.com/FakeFishGames/Barotrauma-development/commit/06a5b01981334819547756884cf657106a964854, fix coming up in the next unstable build.
